modules/dream2nix/package-func/default.nix

```diff
@@ -41,9 +27,7 @@
     // {
       inherit outputs;
       inherit config extendModules;
-      drvPath = throwIfMultiDrvOr outputDrvs.out.drvPath;
-      outPath = throwIfMultiDrvOr outputDrvs.out.outPath;
-      outputName = throwIfMultiDrvOr outputDrvs.out.outputName;
```

Comment:
Shouldn't we keep these checks to prevent people from accidentally mishandling multi-derivation packages?

Reply:
I thought about this for a bit. We can do that with the error demoted to a warning, but it would be hard to remove this warning in the future without introducing surprises. My thinking goes that since the module errors on multi-output derivations as of the status quo anyways, we can document the behavior and do without the guardrails. The change shouldn't affect any current code that depends on this behavior since they have had to explicitly reference a particular output in order to evaluate.